.circleci/config.yml

```diff
@@ -43,108 +43,76 @@ jobs:
     - image: bylexus/apache-php55
     - image: circleci/mysql:5.7
```

Comment:
whil you are at it, remove mysql from this job, and use your image for this job

Reply:
Hmm. Didn't seem to work if I just remove this line.

Reply:
why does rc tag need mysql 🤔. let me see

Reply:
we dont use any mysql capabilities at all. 

we dont even checkout in this job? did it ever work?

Reply:
Oh tag image. My mistake. That's Matt's thing. I'd just let him fix it.

Reply:
No it's not working. Matt has a task to fix it so it'd just let him do it.

Reply:
i think its ok for you to remove mysql from here and add your image. dont need to do anything else. lets make sure everything is uniform

Reply:
I'm actually pretty sure we can use circle's base image here so I'll update it to that.

Reply:
Ok so I removed some stuff I think is not needed.